internal/host/validator.go

```diff
@@ -243,6 +243,35 @@ func (v *validator) hasMinMemory(c *validationContext) ValidationStatus {
 	return boolValue(c.inventory.Memory.PhysicalBytes >= conversions.MibToBytes(c.minRAMMibRequirement))
 }
 
+func (v *validator) hasSupportedPlatform(c *validationContext) ValidationStatus {
+	if c.inventory == nil || c.cluster.Platform.Type == "" {
+		return ValidationPending
+	}
+	hostAvailablePlatforms := common.GetSupportedPlatforms(*c.inventory)
+
+	for _, platform := range *hostAvailablePlatforms {
+		if c.cluster.Platform.Type == platform {
+			return boolValue(true)
```

Comment:
boolValue?

Reply:
This is the convention on this file, it returns `ValidationSuccess`